karma fails to launch firefox on windows

I'm using windows 10, and upon `npm test`, I get below errors.

```

> snabbdom@0.7.3 test C:\Users\Dell\Documents\OpenSource\snabbdom
> karma start

04 09 2019 19:33:52.195:INFO [compiler.karma-typescript]: Compiling project using Typescript 3.3.4000
04 09 2019 19:33:56.420:INFO [compiler.karma-typescript]: Compiled 26 files in 4184 ms.
04 09 2019 19:33:57.271:INFO [bundler.karma-typescript]: Bundled imports for 26 file(s) in 347 ms.
04 09 2019 19:33:58.460:INFO [karma-server]: Karma v3.1.4 server started at http://0.0.0.0:9876/
04 09 2019 19:33:58.464:INFO [launcher]: Launching browsers Chrome, Firefox with concurrency unlimited
04 09 2019 19:33:58.475:INFO [launcher]: Starting browser Chrome
04 09 2019 19:33:58.496:INFO [launcher]: Starting browser Firefox
04 09 2019 19:33:59.664:ERROR [launcher]: Cannot start Firefox

04 09 2019 19:33:59.856:ERROR [launcher]: Firefox stdout:
04 09 2019 19:33:59.862:ERROR [launcher]: Firefox stderr:
04 09 2019 19:34:00.131:INFO [Chrome 76.0.3809 (Windows 10.0.0)]: Connected on socket e0i6ssKrBPlH2bc5AAAA with id 50135268
................................................................................
...................................................................
Chrome 76.0.3809 (Windows 10.0.0): Executed 147 of 147 SUCCESS (1.165 secs / 0.89 secs)
04 09 2019 19:34:03.087:INFO [launcher]: Trying to start Firefox again (1/2).
04 09 2019 19:34:03.119:WARN [temp-dir]: Failed to create a temp dir at C:\Users\Dell\AppData\Local\Temp\karma-16770268
04 09 2019 19:34:07.358:INFO [Firefox 69.0.0 (Windows 10.0.0)]: Connected on socket 5yt0zUBAJgOgBIT9AAAB with id 16770268
................................................................................
................................04 09 2019 19:34:08.726:ERROR [launcher]: Firefox crashed.

04 09 2019 19:34:08.761:ERROR [launcher]: Firefox stdout:
04 09 2019 19:34:08.763:ERROR [launcher]: Firefox stderr:
.....................
Firefox 69.0.0 (Windows 10.0.0) style applies tranform as transition on remove FAILED
        uncaught exception: AssertionError: 1 === 0 (:0)
.............
Firefox 69.0.0 (Windows 10.0.0): Executed 147 of 147 (1 FAILED) (2.19 secs / NaN secs)
TOTAL: 1 FAILED, 293 SUCCESS
npm ERR! Test failed.  See above for more details.
```

I noticed 2 issues.
1. Firefox crashed
2. Assertion failure in Firefox

This issue is specifically for `04 09 2019 19:34:08.726:ERROR [launcher]: Firefox crashed.`

I seem to have a fix for this. PR coming up.

By the way, @mightyiam is mentoring me on this.

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "karma": "^3.0.0",
     "karma-browserstack-launcher": "^1.3.0",
     "karma-chrome-launcher": "^2.2.0",
-    "karma-firefox-launcher": "^1.1.0",
+    "karma-firefox-launcher": "^1.2.0",
     "karma-mocha": "^1.3.0",
     "karma-typescript": "^3.0.13",
     "knuth-shuffle": "^1.0.1",

